### ISSUE TITLE
User Story #8: Set up backend logic to store user data while keeping posts anonymous on the frontend.

### ISSUE THREAD

**Issue by ogometz** [CMU-313/nodebb-f24-aawaa#1]
Backend Configuration for User Anonymity
- Estimated Effort: 7-9  hours
- Dependencies: None
- Team Member Assignments: Owen
- Milestones: Sprint 1
- Acceptance Criteria: User information is securely stored in the backend while posts appear anonymous on the frontend. Verified by posting anonymously and confirming the information remains hidden on the frontend and available in the backend.

**Comment by ogometz** [comment]
This item was not able to be completed in this sprint. Our first mistake was splitting up the work by components within the feature. We realized that with this feature, it was very hard to divide labour by backend vs front end since front end changes are needed to test the back-end modifications. Further, our team made the mistake of assigning this heavy component to just one individual, Owen, who worked on this feature well over the expected number of hours per week for this course, and reached out to his teammates for help later in the sprint. Then, the rest of the team either picked up a task related to the anonymization feature (debugging, UI changes) or picked a new task to work on so we could all have made at least some progress for this sprint. In the end, we made some headway but this task is still left incomplete as of Tuesday evening. For the next sprint, our team is going to ensure that we divide work by feature as opposed to components within a feature, so that other people's work is not blocked by others progress. Furthermore, we will try to ensure that no one persons' feature requires an exorbitant amount of engineering hours to complete. Therefore, this entire feature (anonymization) will be picked back up and owned by Owen, who is confident that given the two weeks of the next sprint will be able to accomplish it.
